server/src/resolvers/getVariantsResolver/utils/liftOver.ts

```diff
@@ -0,0 +1,93 @@
+import { AssemblyId, VariantQueryDataResult } from '../../../types';
+import resolveAssembly from '../utils/resolveAssembly';
+const fs = require('fs/promises');
+const tmpdir = require('os').tmpdir;
+const path = require('path');
+const promiseExec = require('util').promisify(require('child_process').exec);
+
+const createTmpFile = async () => {
+  const filename = Math.random().toString(36).slice(2);
+  const dir = await fs.mkdtemp(path.join(tmpdir(), 'liftover-'));
+  return path.join(dir, filename);
+};
+
+// Get start positions of lifted variants.
+const parseBedStart = (bed: String) =>
```

Comment:
We should add some string validation with regex. It might help others understand what exact string format this string needs to conform to.

Reply:
I am not sure where we would apply string validation with regex. The output of liftOver is 2 BED files: one for lifted variants, one for unlifted variants.  BED file has a specific format: [https://genome.ucsc.edu/FAQ/FAQformat.html](https://genome.ucsc.edu/FAQ/FAQformat.html).

The BED file for unlifted variants has one line of comment starting with "#" preceding each line of variant position, thus, these two functions `parseBedStart`, `parseBedEnd` filter out rows that start with "#".